db/seeds.rb

```diff
@@ -9,3 +9,54 @@
 titles.each do |title|
   Topic.find_or_create_by(title: title)
 end
+
+# user = User.new
```

Comment:
Why these commented lines?

Reply:
Because I have an issue with seeding users, cant seed an user more than once, and i cant use find_or_create_by method either. So gotta comment the lines if im seeding several times, or i would get 'email already taken' error